client/my-sites/site-settings/form-general.jsx

```diff
@@ -151,7 +152,8 @@ const FormGeneral = React.createClass( {
 							valueLink={ this.linkState( 'blogdescription' ) }
 							disabled={ this.state.fetchingSettings }
 							onClick={ this.onRecordEvent( 'Clicked Site Site Tagline Field' ) }
-							onKeyPress={ this.onRecordEventOnce( 'typedTagline', 'Typed in Site Site Tagline Field' ) } />
+							onKeyPress={ this.onRecordEventOnce( 'typedTagline', 'Typed in Site Site Tagline Field' ) }
```

Comment:
Not my code, someone else is tracking it, so we should keep it. Looks changed here in diff only because I moved tag closing from this line to the next one.